### ISSUE TITLE
Develop story idea

### ISSUE THREAD

**Issue by LuizGyro** [LuizGyro/tcc-mac0499#5]

**Comment by grvl** [comment]
Idea: make the protagonist a dog that got lost from it's owner and needs to go through puzzles given by other animals to get back home.

**Comment by LuizGyro** [comment]
That's actually a good idea. Does the dog talk?

**Comment by grvl** [comment]
No, that's part of the character design. He's like a huge lovable but kinda stupid dog. Like a golden.

**Comment by LuizGyro** [comment]
> No, that's part of the character design. He's like a huge lovable but kinda stupid dog. Like a golden.

You should check out how this is coming along. Maybe give me feedback on my poor writing skills.
